README.rst

```diff
@@ -91,6 +91,8 @@ The defintion for each components is:
 - **version**: the version of the package. Optional.
 - **qualifiers**: extra qualifying data for a package such as an OS,
   architecture, a distro, etc. Optional and type-specific.
+  Multiple qualifiers must be separated with "&" and the keys should be
```

Comment:
Do you mind to put that in a separate PR?
I agree that some improvement would be welcomed there, but you definition may conflict with the spec:

In https://github.com/package-url/purl-spec/tree/ef3fa021b54e3e6014bc6d435cc497db030e234b#how-to-build-purl-string-from-its-components we have:
> sort this list of qualifier strings lexicographically 


And in https://github.com/package-url/purl-spec/tree/ef3fa021b54e3e6014bc6d435cc497db030e234b#a-purl-is-a-url 
> purl qualifiers: this maps to a URL query

So all in all IMHO we could in this area avoid reference to sorting and just introduce that qualifiers are `key=value` pairs separated by `&` using the same syntax as a URL query.

And add a mention about sorting being the "canonical" form in the qualifiers definition in https://github.com/package-url/purl-spec/tree/ef3fa021b54e3e6014bc6d435cc497db030e234b#rules-for-each-purl-component ?

Reply:
@pombredanne - a separate PR would be great! I probably should have done that anyway.

Reply:
At this stage the latest spec states:

> This is a query string composed of zero or more key=value pairs each separated by a '&' ampersand. A key and value are separated by the equal '=' character

Therefore I think we are good wrt. the first part of this PR which is about `Multiple qualifiers must be separated with "&"`